test/e2e/test/elasticsearch/steps_mutation.go

```diff
@@ -193,8 +191,13 @@ func (hc *ContinuousHealthCheck) Start() {
 				return
 			case <-ticker.C:
 				ctx, cancel := context.WithTimeout(context.Background(), continuousHealthCheckTimeout)
+				client, err := hc.esClientFactory()
```

Comment:
Do we need the factory vs. just calling `NewElasticsearchClient` here?

Reply:
Well either that or we have to pass the k8s Client to the struct

Reply:
ah that's why, got it! Fine with me then :)